### PR TITLE
Add handling of live override mode

### PR DESCRIPTION
**Pull request by drinfernoo**
This adds `lor` as a field in the device state, as it is only applicable to the entire device, and not to each individual segment.

### CODE REVIEW

**Comment by drinfernoo** [comment]
Hmm, why did this fail? 🤔

**Comment by drinfernoo** [comment]
Am I passing the wrong responses to the test framework? They looked right to me...

File: src/wled/wled.py [hunk 1]
```diff
@@ -372,6 +372,16 @@ async def preset(self, preset: int) -> None:
         """
         await self.request("state", method="POST", data={"ps": preset})
 
+    async def live(self, live: int) -> None:
```
Comment (frenck) [inline src/wled/wled.py:375]: Should we accept a ENUM with the different modes?

Comment (drinfernoo) [reply src/wled/wled.py:375]: That does seem like a better option! I can implement that later this evening probably 👍

I'm still not sure why those tests aren't passing... do you have any idea?

Comment (frenck) [reply src/wled/wled.py:375]: Haven't looked yet, I don't see an obvious reason on first sight.

**Comment by frenck** [comment]
There is nothing to review for me at this point, as the CI is failing. Please make sure the CI is passed before requesting reviews.

**Comment by drinfernoo** [comment]
My bad, I'm at a loss for why those tests are failing 😕

**Comment by drinfernoo** [comment]
Hey! That's... better 😅

**Comment by drinfernoo** [comment]
I don't appear to be able to add a label, but everything else is passing here 👍

**Review by frenck — APPROVED** [review]
Thanks, @drinfernoo 👍

**Comment by drinfernoo** [comment]
🎉🎉

**Pull request merged**